qiskit/ignis/verification/tomography/interface.py

```diff
@@ -0,0 +1,149 @@
+                'ideal_psi': ideal statevector
+                'fidelity' the fidelity between rho and ideal_psi
+       """
+
+    if backend is None:
+        backend = Aer.get_backend('qasm_simulator')
+    tomography_circuits = state_tomography_circuits(circuit, measured_qubits)
+    job = qiskit.execute(tomography_circuits, backend, shots=shots)
+    data_results = tomography_data(job.result(), tomography_circuits)
+    data, basis, _ = fitter_data(data_results)
+    rho = None
+    if fitter == 'cvx':
+        try:
+            rho = state_cvx_fit(data, basis)
+        except RuntimeError as e:
+            print("CVX run failed: {}, attempting MLE fit".format(e))
+    if fitter == 'mle' or rho is None:
+        rho = state_mle_fit(data, basis)
+
+    if ideal is False and fidelity is False:
+        return rho
+
+    result = {'rho': rho}
+    if ideal:
+        job = qiskit.execute(circuit, Aer.get_backend('statevector_simulator'))
+        ideal_psi = job.result().get_statevector(circuit)
+        result['ideal_psi'] = ideal_psi
+        if fidelity:
+            fidelity_value = state_fidelity(ideal_psi, rho)
+            result['fidelity'] = fidelity_value
+
+    return result
+
+
+def perform_process_tomography(circuit, measured_qubits,
+                               backend=None,
+                               fitter='cvx',
+                               ideal=True,
+                               fidelity=True,
+                               shots=4000, ):
+    """
+       Run the full process tomography flow:
+       1) Generate tomography circuits.
+       2) Run the circuits on the given backend
+       3) Extract from the results a fitter-friendly representation
+       4) Run fitter to obtain choi matrix
+       5) (optional) create an ideal result using a unitary simulator
+
+       Args:
+           circuit: the circuit representing the process to measure
+           measured_qubits: the qubits to be measured
+           backend: the backend on which the circuit is ran
+           fitter: the fitter used to obtain the choi matrix
+           ideal: whether to compute an ideal choi matrix
+           fidelity: whether to compute the fidelity between the tomography
+                     result and the ideal result
+           shots: how many shots to perform with the backend
+                  on each measurement
+
+       Returns:
+            if ideal and fidelity are False, returns the density operator
+            otherwise returns a dictionary with keys
+                'rho': the density operator
+                'ideal_psi': ideal statevector
+                'fidelity' the fidelity between rho and ideal_psi
+       """
+
+    if backend is None:
+        backend = Aer.get_backend('qasm_simulator')
+    circuits = process_tomography_circuits(circuit, measured_qubits)
+    job = qiskit.execute(circuits, backend, shots=shots)
+    tomography_data_results = tomography_data(job.result(), circuits)
```

Comment:
Same thing here but `ProcessTomographyFitter`

Reply:
This was fixed